R/data.table.R

```diff
@@ -1884,7 +1886,7 @@ as.matrix.data.table = function(x, rownames=NULL, rownames.value=NULL, ...) {
   non.numeric = non.atomic = FALSE
   all.logical = TRUE
   for (j in seq_len(p)) {
-    if (is.ff(X[[j]])) X[[j]] = X[[j]][]   # to bring the ff into memory, since we need to create a matrix in memory
+    if (is.ff(X[[j]])) X[[j]] = X[[j]][]   # nocov to bring the ff into memory, since we need to create a matrix in memory
```

Comment:
we don't have `ff` in `Suggests`, so `nocov`. We have `is.ff` as a simple wrapper, so technically we could construct some object and force `ff` class to it & test but that's not really a test of what this branch is supposed to do. So `nocov` unless we want to add `ff` to `Suggests`, or if you think we should add `ff` to the test packages.